build-source-tarball.sh

```diff
@@ -129,6 +130,12 @@ do
     fi
 done
 
+for built_package in $(find $SCRIPT_ROOT/bin/obj/x64/Release/blob-feed/packages/ -name '*.nupkg')
+do
+    echo "copying $built_package to $TARBALL_ROOT/prebuilt/source-built/"
```

Comment:
I think you should add an `echo` before the `for` saying what these copies are for.

Reply:
I'll add an echo but printing each package was just for debugging, I'll remove it.